src/Aspire.Dashboard/Configuration/DashboardOptions.cs

```diff
@@ -89,18 +96,22 @@ public byte[] GetPrimaryApiKeyBytes()
 
     internal bool TryParseOptions([NotNullWhen(false)] out string? errorMessage)
     {
-        if (string.IsNullOrEmpty(EndpointUrl))
+        if (string.IsNullOrEmpty(GrpcEndpointUrl) && string.IsNullOrEmpty(HttpEndpointUrl))
         {
-            errorMessage = $"OTLP endpoint URL is not configured. Specify a {DashboardConfigNames.DashboardOtlpUrlName.EnvVarName} value.";
+            errorMessage = $"Both OTLP gRPC endpoint URL and OTLP HTTP endpoint URL are not configured. Specify either a {DashboardConfigNames.DashboardOtlpGrpcUrlName.EnvVarName} or {DashboardConfigNames.DashboardOtlpHttpUrlName.EnvVarName} value.";
```

Comment:
I found this a bit hard to parse. What about this?

```suggestion
            errorMessage = $"Neither gRPC or HTTP endpoint URL are configured for OTLP. Specify either a {DashboardConfigNames.DashboardOtlpGrpcUrlName.EnvVarName} or {DashboardConfigNames.DashboardOtlpHttpUrlName.EnvVarName} value.";
```